style.css

```diff
@@ -121,7 +120,7 @@ h3 {
   position: absolute;
   background-color: #f1f1f1;
   min-width: 160px;
-  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  box-shadow: 0px 8px 16px 0 rgba(0, 0, 0, 0.2);
```

Comment:
zero value does not require to have units. It will provide uniformity in the code. 
please refer to the following link: https://www.dunkman.me/blog/2017/css-zero-length-units/
Please let me know what you think.